doc/index.rst

```diff
@@ -114,3 +114,4 @@ Core Lightning Documentation
    lightning-withdraw <lightning-withdraw.7.md>
    lightningd <lightningd.8.md>
    lightningd-config <lightningd-config.5.md>
+   lightningd-json <lightningd-json.7.md>
```

Comment:
This file does not exist.

Reply:
Yes, was on a different branch, it didn't get re-regenerated when I switched.  Gah! thanks